pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -451,45 +470,57 @@ func (s *nsmgrSuite) Test_PassThroughRemoteUsecase() {
 
 func (s *nsmgrSuite) Test_PassThroughLocalUsecase() {
 	t := s.T()
-	const nsesCount = 7
+	const nsesCount = 5
```

Comment:
Do we need this count for any reason to be changed?

Reply:
not needed after review fixes, returned old value